packages/indy-vdr/src/anoncreds/IndyVdrAnonCredsRegistry.ts

```diff
@@ -287,8 +287,9 @@ export class IndyVdrAnonCredsRegistry implements AnonCredsRegistry {
       }
 
       // Format the schema id based on the type of the credential definition id
+      const unqualifiedDid = schema.schema.schemaId.split(':')[0]
       const schemaId = credentialDefinitionId.startsWith('did:indy')
-        ? getDidIndySchemaId(pool.indyNamespace, namespaceIdentifier, schema.schema.name, schema.schema.version)
+        ? getDidIndySchemaId(pool.indyNamespace, unqualifiedDid, schema.schema.name, schema.schema.version)
```

Comment:
If my comment below regarding returning the correct issuerId in `fetchIndySchemaWithSeqNo` is correct, I think we could simply `schema.schema.issuerId`, so there will not be need of creating the unqualifiedDid variable.

Reply:
Yes right , I will make the necessary changes.

Reply:
I have made the necessary changes